api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -135,4 +136,14 @@ public void validateInstitutionalAffiliation(Profile profile) {
       throw new BadRequestException(msg);
     }
   }
+
+  public void adminUpdateProfile(Profile updatedProfile) {
```

Comment:
ProfileController.adminUpdateProfile() will be pushed down to here as part of RW-5104